### PR TITLE
Pipeline test

### PR DESCRIPTION
**Pull request by madhuintel**

### CODE REVIEW

**Review by jedwards-habana — CHANGES_REQUESTED** [review]
I think the pipeline files should have a common naming format. Something like pr-monday-connector.yaml. I think we can drop the unit testtest on the end. Also, lets use '-' instead of underscores.

File: .github/workflows/monday_connector_unittests.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  push:
+    branches:
+      - pipeline-test
+    paths: 
+      - monday_connector/**
+      
+  pull_request:
+
+jobs:
+  build:
```
Comment (jedwards-habana) [inline .github/workflows/monday_connector_unittests.yaml:13]: This isn't a build step. It is a 'unittest' step. This text will show as 'build' in the step automation and it should be consistent with what is happening.

File: .github/workflows/snowflake_unittests.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  push:
+    branches:
+      - pipeline-test
+    paths: 
+      - snowflake_connector/**
+      
+  pull_request:
+
+jobs:
+  build:
```
Comment (jedwards-habana) [inline .github/workflows/snowflake_unittests.yaml:13]: unittest step.

File: .github/workflows/wiki_unittests.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  push:
+    branches:
+      - pipeline-test
+    paths: 
+      - wikipedia_connector/**
+      
+  pull_request:
+
+jobs:
+  build:
```
Comment (jedwards-habana) [inline .github/workflows/wiki_unittests.yaml:13]: unittest step.

**Review by jedwards-habana — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-snowflake-connector.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: Unit Tests for Snowflake Connector 
+
+env:
+    TEST_DIRECTORY: snowflake_connector
+
+on:
+  push:
+    branches:
+      - main
+      - pipeline-test
```
[the remark this answers is not in the document]
Comment (madhuintel) [reply .github/workflows/pr-snowflake-connector.yaml]: As discussed, I have updated all the files.

**Review by jedwards-habana — APPROVED** [review]
LGTM.

**Pull request merged**